context/src/env/validator.rs

```diff
@@ -76,21 +88,18 @@ pub enum EnvValidationIssueSubOptimal {
     CIInfoJobURLTooShort(String),
     #[error("CI info job URL too long, truncated to {}", MAX_FIELD_LEN)]
     CIInfoJobURLTooLong(String),
+    #[error("CI info is classified as a PR, but has not PR number")]
+    CIInfoPRNumberMissing,
+    #[error("CI info has a PR number, but branch is not classified as a PR")]
+    CIInfoPRNumberConflictsWithBranchClass,
     #[error("CI info title too short")]
     CIInfoTitleTooShort(String),
     #[error("CI info title too long, truncated to {}", MAX_FIELD_LEN)]
     CIInfoTitleTooLong(String),
 }
 
 #[derive(Error, Debug, Clone, PartialEq, Eq)]
-pub enum EnvValidationIssueInvalid {
-    #[error("CI info branch name too short")]
-    CIInfoBranchNameTooShort(String),
-    #[error("CI info is classified as a PR, but has not PR number")]
-    CIInfoPRNumberMissing,
-    #[error("CI info has a PR number, but branch is not classified as a PR")]
-    CIInfoPRNumberConflictsWithBranchClass,
-}
+pub enum EnvValidationIssueInvalid {}
```

Comment:
Is this used anymore? Should we just delete it?

Reply:
Not currently used, but I'm fine with leaving it so that if we want to add an invalid issue of this type in the near future, the code is already equipped to handle it